CONTRIBUTING.md

```diff
@@ -100,7 +100,7 @@ Ideally, you should use a dedicated MarkLogic instance for this purpose altogeht
 
 On the MarkLogic instance, create an HTTP server in MarkLogic with the following non-default properties:
 
-- server name: JSdebugTestServer
+- server name: localhost
```

Comment:
If this should be a host name instead of a server name, can you rename this to "host name" (or just "host")?

Reply:
ack. Clearly I was sleepy. The original is correct. I will revert this change.